src/data/markdown/translated-guides/en/01 Get started/02 Installation/01 Troubleshooting.md

```diff
@@ -47,7 +52,24 @@ gpg: keyserver receive failed: No dirmngr
 ```
 
 This happens if it's the first time that the user runs `gpg` , so the directory `/root/.gnupg/` doesn't exist yet.
-To create the directory, run `sudo gpg -k` and try to import the key again.
+
+
+To create the directory, run `sudo gpg -k` and try to import the key again:
+
+```bash
+# list keys to ensure the presence of the `.gnupg` directory
+sudo gpg -k
+
+# delete existing key
+sudo apt-key del k6
+
+# import the key the recommended way
+sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
+
+# update the repository
+echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+sudo apt-get update
```

Comment:
I would remove these instructions from the Troubleshooting page. This page should only list issues and suggestions to fix them, leaving the actual installation instructions to the Installation page. That way we don't have to remember to update both pages when the instructions change.

These are also repeated above in the `apt-key` deprecation section, and I'd remove it from there as well.